tests/gtest/aviflosslesstest.cc

```diff
@@ -12,86 +14,146 @@ namespace {
 // Used to pass the data folder path to the GoogleTest suites.
 const char* data_path = nullptr;
 
+// Read image losslessly, using identiy MC.
+ImagePtr ReadImageLossless(const std::string& path,
+                           avifPixelFormat requested_format,
+                           avifBool ignore_icc) {
+  ImagePtr image(avifImageCreateEmpty());
+  image->matrixCoefficients = AVIF_MATRIX_COEFFICIENTS_IDENTITY;
+  if (!image ||
+      avifReadImage(path.c_str(), requested_format, /*requested_depth=*/0,
+                    /*chroma_downsampling=*/AVIF_CHROMA_DOWNSAMPLING_AUTOMATIC,
+                    ignore_icc, /*ignore_exif=*/false, /*ignore_xmp=*/false,
+                    /*allow_changing_cicp=*/false, /*ignore_gain_map=*/false,
+                    AVIF_DEFAULT_IMAGE_SIZE_LIMIT, image.get(),
+                    /*outDepth=*/nullptr, /*sourceTiming=*/nullptr,
+                    /*frameIter=*/nullptr) == AVIF_APP_FILE_FORMAT_UNKNOWN) {
+    return nullptr;
+  }
+  return image;
+}
+
+class LosslessTest
```

Comment:
Add a comment about what the parameters are. Tuples are convenient in tests, but aren't self documenting like structs.

Reply:
This was always up for debate: they are written four lines below. I agree that it is useful to repeat them when the list of elements gets long. Do you mean we should always do it for consistency?

Reply:
> This was always up for debate: they are written four lines below. I agree that it is useful to repeat them when the list of elements gets long. Do you mean we should always do it for consistency?

With tuples if the types don't give enough information I think it's a good practice.